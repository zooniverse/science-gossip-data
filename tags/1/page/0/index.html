<!doctype html>
<html lang="en">
  <head>
  
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta property="og:type" content="website" />
    <meta property="og:site_name" content="Science Gossip Data" />
    <meta property='og:url' content="https://zooniverse.github.io/science-gossip-data/tags/1/page/0/" />
    <meta property='og:title' content="1 Page 0" />
    
    
    <meta property='og:image' content="http://zooniverse-static.s3.amazonaws.com/www.sciencegossip.org/subjects/thumb/54f43a86efc50104c3002e83.jpg" />
    <meta name='twitter:image' content="http://zooniverse-static.s3.amazonaws.com/www.sciencegossip.org/subjects/thumb/54f43a86efc50104c3002e83.jpg" />
  
    <meta name="twitter:card" content="summary" />
    <meta name="twitter:creator" content="@the_zooniverse" />
    <meta name="twitter:site" content="@the_zooniverse" />
    <title>1 Page 0</title>
  
  <link rel="stylesheet" href="https://unpkg.com/open-props"/>
  <link rel="stylesheet" href="https://unpkg.com/open-props/normalize.min.css"/>
  <link rel="stylesheet" href="/science-gossip-data/css/main.css" />
  </head>
  <body>
    <h1>1</h1>
<nav aria-label="Page">
  <ul>
  <li>
      <a
      class="current"
        href="../0/"
      >
        0
      </a>
    </li>
  </ul>
</nav>
<ul class="thumbnails">

  <li>
    <a href="../../../../groups/hardwickesscien/subjects/ASC00004s8">
      
    <img
      loading=lazy
      
      alt="Subject ASC00004s8"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/54f43a86efc50104c3002e83.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/magazineofnatura/subjects/ASC0000g1q">
      
    <img
      loading=lazy
      
      alt="Subject ASC0000g1q"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/54f43af2efc50104c30059f5.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/magazineofnatura/subjects/ASC0000hd6">
      
    <img
      loading=lazy
      
      alt="Subject ASC0000hd6"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/54f43ad3efc50104c3004d71.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/quarterlyjourna/subjects/ASC00013vc">
      
    <img
      loading=lazy
      
      alt="Subject ASC00013vc"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/54f4be6c30017b04c902bd2f.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/journalofquekett/subjects/ASC0002zzd">
      
    <img
      loading=lazy
      
      alt="Subject ASC0002zzd"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/55e96b2fa71c52004100a8fb.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/journalofquekett/subjects/ASC00030s3">
      
    <img
      loading=lazy
      
      alt="Subject ASC00030s3"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/55e96b3ea71c52004100b133.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/the-geologist-a-popular-monthly-magazine-of-geology/subjects/ASC0003w7k">
      
    <img
      loading=lazy
      
      alt="Subject ASC0003w7k"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/56c48b10bcdc20004200873b.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/journal-of-botany-british-and-foreign/subjects/ASC0003s30">
      
    <img
      loading=lazy
      
      alt="Subject ASC0003s30"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/56c48aadbcdc200042005d15.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/journal-of-botany-british-and-foreign/subjects/ASC0003p69">
      
    <img
      loading=lazy
      
      alt="Subject ASC0003p69"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/56c48a69bcdc200042003f9f.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/journal-of-botany-british-and-foreign/subjects/ASC0003nzl">
      
    <img
      loading=lazy
      
      alt="Subject ASC0003nzl"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/56c48a4bbcdc200042003329.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/the-journal-of-horticulture-cottage-gardener-and-country-gentlemen/subjects/ASC0003mj1">
      
    <img
      loading=lazy
      
      alt="Subject ASC0003mj1"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/56c48a2abcdc20004200250d.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/the-journal-of-horticulture-cottage-gardener-and-country-gentlemen/subjects/ASC0003mr2">
      
    <img
      loading=lazy
      
      alt="Subject ASC0003mr2"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/56c48a2fbcdc200042002755.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/the-journal-of-horticulture-cottage-gardener-and-country-gentlemen/subjects/ASC0003ld5">
      
    <img
      loading=lazy
      
      alt="Subject ASC0003ld5"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/56c48a0fbcdc200042001989.jpg"
    />
  
    </a>
  </li>

</ul>
<nav aria-label="Page">
  <ul>
  <li>
      <a
      class="current"
        href="../0/"
      >
        0
      </a>
    </li>
  </ul>
</nav>
  </body>
</html>