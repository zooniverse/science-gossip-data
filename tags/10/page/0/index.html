<!doctype html>
<html lang="en">
  <head>
  
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta property="og:type" content="website" />
    <meta property="og:site_name" content="Science Gossip Data" />
    <meta property='og:url' content="https://zooniverse.github.io/science-gossip-data/tags/10/page/0/" />
    <meta property='og:title' content="10 Page 0" />
    
    <meta name="description" content="Illustrated pages tagged with 10" />
    <meta name='twitter:description' content="Illustrated pages tagged with 10" />
    <meta property='og:description' content="Illustrated pages tagged with 10" />
  
    
    <meta property='og:image' content="http://zooniverse-static.s3.amazonaws.com/www.sciencegossip.org/subjects/thumb/54f4b8f930017b04c9005b1d.jpg" />
    <meta name='twitter:image' content="http://zooniverse-static.s3.amazonaws.com/www.sciencegossip.org/subjects/thumb/54f4b8f930017b04c9005b1d.jpg" />
  
    <meta name="twitter:card" content="summary" />
    <meta name="twitter:creator" content="@the_zooniverse" />
    <meta name="twitter:site" content="@the_zooniverse" />
    <title>10 Page 0</title>
  
  <link rel="stylesheet" href="https://unpkg.com/open-props"/>
  <link rel="stylesheet" href="https://unpkg.com/open-props/normalize.min.css"/>
  <link rel="stylesheet" href="/css/main.css" />
  </head>
  <body>
    <h1>10</h1>
<nav aria-label="Page">
  <ul>
  <li>
      <a
      class="current"
        href="../0/"
      >
        0
      </a>
    </li>
  </ul>
</nav>
<ul class="thumbnails">

  <li>
    <a href="../../../../groups/wiltshirearchaeo/subjects/ASC0000pv3">
      
    <img
      loading=lazy
      
      alt="Subject ASC0000pv3"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/54f4b8f930017b04c9005b1d.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/journalofquekett/subjects/ASC0002zzd">
      
    <img
      loading=lazy
      
      alt="Subject ASC0002zzd"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/55e96b2fa71c52004100a8fb.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/journal-of-botany-british-and-foreign/subjects/ASC0003s30">
      
    <img
      loading=lazy
      
      alt="Subject ASC0003s30"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/56c48aadbcdc200042005d15.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/the-journal-of-horticulture-cottage-gardener-and-country-gentlemen/subjects/ASC0003l9f">
      
    <img
      loading=lazy
      
      alt="Subject ASC0003l9f"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/56c48a0bbcdc2000420017b9.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/the-journal-of-horticulture-cottage-gardener-and-country-gentlemen/subjects/ASC0003leu">
      
    <img
      loading=lazy
      
      alt="Subject ASC0003leu"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/56c48a0fbcdc200042001979.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/the-journal-of-horticulture-cottage-gardener-and-country-gentlemen/subjects/ASC0003lhr">
      
    <img
      loading=lazy
      
      alt="Subject ASC0003lhr"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/56c48a11bcdc200042001a2b.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/the-journal-of-horticulture-cottage-gardener-and-country-gentlemen/subjects/ASC0003kz3">
      
    <img
      loading=lazy
      
      alt="Subject ASC0003kz3"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/56c48a04bcdc2000420014ff.jpg"
    />
  
    </a>
  </li>

</ul>
<nav aria-label="Page">
  <ul>
  <li>
      <a
      class="current"
        href="../0/"
      >
        0
      </a>
    </li>
  </ul>
</nav>
  </body>
</html>