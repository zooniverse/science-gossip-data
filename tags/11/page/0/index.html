<!doctype html>
<html lang="en">
  <head>
  
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta property="og:type" content="website" />
    <meta property="og:site_name" content="Science Gossip Data" />
    <meta property='og:url' content="https://zooniverse.github.io/science-gossip-data/tags/11/page/0/" />
    <meta property='og:title' content="11 Page 0" />
    
    
    <meta property='og:image' content="http://zooniverse-static.s3.amazonaws.com/www.sciencegossip.org/subjects/thumb/54f4b8f930017b04c9005b1d.jpg" />
    <meta name='twitter:image' content="http://zooniverse-static.s3.amazonaws.com/www.sciencegossip.org/subjects/thumb/54f4b8f930017b04c9005b1d.jpg" />
  
    <meta name="twitter:card" content="summary" />
    <meta name="twitter:creator" content="@the_zooniverse" />
    <meta name="twitter:site" content="@the_zooniverse" />
    <title>11 Page 0</title>
  
  <link rel="stylesheet" href="https://unpkg.com/open-props"/>
  <link rel="stylesheet" href="https://unpkg.com/open-props/normalize.min.css"/>
  <link rel="stylesheet" href="/science-gossip-data/css/main.css" />
  </head>
  <body>
    <h1>11</h1>
<nav aria-label="Page">
  <ul>
  <li>
      <a
      class="current"
        href="../0/"
      >
        0
      </a>
    </li>
  </ul>
</nav>
<ul class="thumbnails">

  <li>
    <a href="../../../../groups/wiltshirearchaeo/subjects/ASC0000pv3">
      
    <img
      loading=lazy
      
      alt="Subject ASC0000pv3"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/54f4b8f930017b04c9005b1d.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/wiltshirearchaeo/subjects/ASC0000vsi">
      
    <img
      loading=lazy
      
      alt="Subject ASC0000vsi"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/54f4b86830017b04c9001d6b.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/journalofquekett/subjects/ASC0002zzd">
      
    <img
      loading=lazy
      
      alt="Subject ASC0002zzd"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/55e96b2fa71c52004100a8fb.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/journal-of-botany-british-and-foreign/subjects/ASC0003s30">
      
    <img
      loading=lazy
      
      alt="Subject ASC0003s30"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/56c48aadbcdc200042005d15.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/the-journal-of-horticulture-cottage-gardener-and-country-gentlemen/subjects/ASC0003kri">
      
    <img
      loading=lazy
      
      alt="Subject ASC0003kri"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/56c48a00bcdc20004200133f.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/the-journal-of-horticulture-cottage-gardener-and-country-gentlemen/subjects/ASC0003kif">
      
    <img
      loading=lazy
      
      alt="Subject ASC0003kif"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/56c489f9bcdc20004200104d.jpg"
    />
  
    </a>
  </li>

</ul>
<nav aria-label="Page">
  <ul>
  <li>
      <a
      class="current"
        href="../0/"
      >
        0
      </a>
    </li>
  </ul>
</nav>
  </body>
</html>