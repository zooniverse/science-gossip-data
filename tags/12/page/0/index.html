<!doctype html>
<html lang="en">
  <head>
  
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta property="og:type" content="website" />
    <meta property="og:site_name" content="Science Gossip Data" />
    <meta property='og:url' content="https://zooniverse.github.io/science-gossip-data/tags/12/page/0/" />
    <meta property='og:title' content="12 Page 0" />
    
    <meta name="description" content="Illustrated pages tagged with 12" />
    <meta name='twitter:description' content="Illustrated pages tagged with 12" />
    <meta property='og:description' content="Illustrated pages tagged with 12" />
  
    
    <meta property='og:image' content="http://zooniverse-static.s3.amazonaws.com/www.sciencegossip.org/subjects/thumb/54f43afaefc50104c3005d2b.jpg" />
    <meta name='twitter:image' content="http://zooniverse-static.s3.amazonaws.com/www.sciencegossip.org/subjects/thumb/54f43afaefc50104c3005d2b.jpg" />
  
    <meta name="twitter:card" content="summary" />
    <meta name="twitter:creator" content="@the_zooniverse" />
    <meta name="twitter:site" content="@the_zooniverse" />
    <title>12 Page 0</title>
  
  <link rel="stylesheet" href="https://unpkg.com/open-props"/>
  <link rel="stylesheet" href="https://unpkg.com/open-props/normalize.min.css"/>
  <link rel="stylesheet" href="/science-gossip-data/css/main.css" />
  </head>
  <body>
    <h1>12</h1>
<nav aria-label="Page">
  <ul>
  <li>
      <a
      class="current"
        href="../0/"
      >
        0
      </a>
    </li>
  </ul>
</nav>
<ul class="thumbnails">

  <li>
    <a href="../../../../groups/magazineofnatura/subjects/ASC0000frr">
      
    <img
      loading=lazy
      
      alt="Subject ASC0000frr"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/54f43afaefc50104c3005d2b.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/wiltshirearchaeo/subjects/ASC0000pv3">
      
    <img
      loading=lazy
      
      alt="Subject ASC0000pv3"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/54f4b8f930017b04c9005b1d.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/wiltshirearchaeo/subjects/ASC0000tyc">
      
    <img
      loading=lazy
      
      alt="Subject ASC0000tyc"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/54f4b89830017b04c900326d.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/journalofquekett/subjects/ASC0002zzd">
      
    <img
      loading=lazy
      
      alt="Subject ASC0002zzd"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/55e96b2fa71c52004100a8fb.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/journal-of-botany-british-and-foreign/subjects/ASC0003s30">
      
    <img
      loading=lazy
      
      alt="Subject ASC0003s30"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/56c48aadbcdc200042005d15.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/the-journal-of-horticulture-cottage-gardener-and-country-gentlemen/subjects/ASC0003mp0">
      
    <img
      loading=lazy
      
      alt="Subject ASC0003mp0"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/56c48a2fbcdc200042002711.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/the-journal-of-horticulture-cottage-gardener-and-country-gentlemen/subjects/ASC0003l5t">
      
    <img
      loading=lazy
      
      alt="Subject ASC0003l5t"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/56c48a09bcdc200042001727.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/the-journal-of-horticulture-cottage-gardener-and-country-gentlemen/subjects/ASC0003ldw">
      
    <img
      loading=lazy
      
      alt="Subject ASC0003ldw"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/56c48a0ebcdc2000420018f9.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/the-journal-of-horticulture-cottage-gardener-and-country-gentlemen/subjects/ASC0003koc">
      
    <img
      loading=lazy
      
      alt="Subject ASC0003koc"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/56c489fdbcdc200042001237.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/the-journal-of-horticulture-cottage-gardener-and-country-gentlemen/subjects/ASC0003kc8">
      
    <img
      loading=lazy
      
      alt="Subject ASC0003kc8"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/56c489f5bcdc200042000e93.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/the-journal-of-horticulture-cottage-gardener-and-country-gentlemen/subjects/ASC0003kse">
      
    <img
      loading=lazy
      
      alt="Subject ASC0003kse"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/56c48a00bcdc20004200133b.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/the-journal-of-horticulture-cottage-gardener-and-country-gentlemen/subjects/ASC0003kn0">
      
    <img
      loading=lazy
      
      alt="Subject ASC0003kn0"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/56c489fbbcdc20004200117b.jpg"
    />
  
    </a>
  </li>

</ul>
<nav aria-label="Page">
  <ul>
  <li>
      <a
      class="current"
        href="../0/"
      >
        0
      </a>
    </li>
  </ul>
</nav>
  </body>
</html>