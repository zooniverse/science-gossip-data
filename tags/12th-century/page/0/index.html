<!doctype html>
<html lang="en">
  <head>
  
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="view-transition" content="same-origin" />
    <meta property="og:type" content="website" />
    <meta property="og:site_name" content="Science Gossip Data" />
    <meta property='og:url' content="https://zooniverse.github.io/science-gossip-data/tags/12th-century/page/0/" />
    <meta property='og:title' content="12th century Page 0" />
    
    <meta name="description" content="Illustrated pages tagged with 12th century" />
    <meta name='twitter:description' content="Illustrated pages tagged with 12th century" />
    <meta property='og:description' content="Illustrated pages tagged with 12th century" />
  
    
    <meta property='og:image' content="http://zooniverse-static.s3.amazonaws.com/www.sciencegossip.org/subjects/thumb/54f4b93230017b04c9007499.jpg" />
    <meta name='twitter:image' content="http://zooniverse-static.s3.amazonaws.com/www.sciencegossip.org/subjects/thumb/54f4b93230017b04c9007499.jpg" />
  
    <meta name="twitter:card" content="summary" />
    <meta name="twitter:creator" content="@the_zooniverse" />
    <meta name="twitter:site" content="@the_zooniverse" />
    <title>12th century Page 0</title>
  
  <link rel="stylesheet" href="https://unpkg.com/open-props"/>
  <link rel="stylesheet" href="https://unpkg.com/open-props/normalize.min.css"/>
  <link rel="stylesheet" href="/science-gossip-data/css/main.css" />
  </head>
  <body>
    <h1>12th century</h1>
<nav aria-label="Page">
  <ul>
  <li>
      <a
      class="current"
        href="../0/"
      >
        0
      </a>
    </li>
  </ul>
</nav>
<ul class="thumbnails">

  <li>
    <a href="../../../../groups/wiltshirearchaeo/subjects/ASC0000o5c">
      
    <img
      loading=lazy
      
      alt="Subject ASC0000o5c"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/54f4b93230017b04c9007499.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/wiltshirearchaeo/subjects/ASC0000oz9">
      
    <img
      loading=lazy
      
      alt="Subject ASC0000oz9"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/54f4b90a30017b04c900627f.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/wiltshirearchaeo/subjects/ASC0000pgh">
      
    <img
      loading=lazy
      
      alt="Subject ASC0000pgh"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/54f4b8fa30017b04c9005b55.jpg"
    />
  
    </a>
  </li>

</ul>
<nav aria-label="Page">
  <ul>
  <li>
      <a
      class="current"
        href="../0/"
      >
        0
      </a>
    </li>
  </ul>
</nav>
  </body>
</html>