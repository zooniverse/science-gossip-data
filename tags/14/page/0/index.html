<!doctype html>
<html lang="en">
  <head>
  
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta property="og:type" content="website" />
    <meta property="og:site_name" content="Science Gossip Data" />
    <meta property='og:url' content="https://zooniverse.github.io/science-gossip-data/tags/14/page/0/" />
    <meta property='og:title' content="14 Page 0" />
    
    
    <meta property='og:image' content="http://zooniverse-static.s3.amazonaws.com/www.sciencegossip.org/subjects/thumb/54f4b90330017b04c9005f3d.jpg" />
    <meta name='twitter:image' content="http://zooniverse-static.s3.amazonaws.com/www.sciencegossip.org/subjects/thumb/54f4b90330017b04c9005f3d.jpg" />
  
    <meta name="twitter:card" content="summary" />
    <meta name="twitter:creator" content="@the_zooniverse" />
    <meta name="twitter:site" content="@the_zooniverse" />
    <title>14 Page 0</title>
  
  <link rel="stylesheet" href="https://unpkg.com/open-props"/>
  <link rel="stylesheet" href="https://unpkg.com/open-props/normalize.min.css"/>
  <link rel="stylesheet" href="/science-gossip-data/css/main.css" />
  </head>
  <body>
    <h1>14</h1>
<nav aria-label="Page">
  <ul>
  <li>
      <a
      class="current"
        href="../0/"
      >
        0
      </a>
    </li>
  </ul>
</nav>
<ul class="thumbnails">

  <li>
    <a href="../../../../groups/wiltshirearchaeo/subjects/ASC0000p70">
      
    <img
      loading=lazy
      
      alt="Subject ASC0000p70"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/54f4b90330017b04c9005f3d.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/wiltshirearchaeo/subjects/ASC0000tyc">
      
    <img
      loading=lazy
      
      alt="Subject ASC0000tyc"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/54f4b89830017b04c900326d.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/gardenersmagazin2/subjects/ASC0002v4f">
      
    <img
      loading=lazy
      
      alt="Subject ASC0002v4f"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/5527aff1a092de04cc00601b.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/gardenersmagazin2/subjects/ASC0002os7">
      
    <img
      loading=lazy
      
      alt="Subject ASC0002os7"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/5527af7ea092de04cc00204d.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/journalofquekett/subjects/ASC0002zzd">
      
    <img
      loading=lazy
      
      alt="Subject ASC0002zzd"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/55e96b2fa71c52004100a8fb.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/journal-of-botany-british-and-foreign/subjects/ASC0003s30">
      
    <img
      loading=lazy
      
      alt="Subject ASC0003s30"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/56c48aadbcdc200042005d15.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/the-journal-of-horticulture-cottage-gardener-and-country-gentlemen/subjects/ASC0003l53">
      
    <img
      loading=lazy
      
      alt="Subject ASC0003l53"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/56c48a08bcdc20004200169d.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/the-journal-of-horticulture-cottage-gardener-and-country-gentlemen/subjects/ASC0003kgu">
      
    <img
      loading=lazy
      
      alt="Subject ASC0003kgu"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/56c489f9bcdc20004200102f.jpg"
    />
  
    </a>
  </li>

</ul>
<nav aria-label="Page">
  <ul>
  <li>
      <a
      class="current"
        href="../0/"
      >
        0
      </a>
    </li>
  </ul>
</nav>
  </body>
</html>