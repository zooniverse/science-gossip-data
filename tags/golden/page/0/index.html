<!doctype html>
<html lang="en">
  <head>
  
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta property="og:type" content="website" />
    <meta property="og:site_name" content="Science Gossip Data" />
    <meta property='og:url' content="https://zooniverse.github.io/science-gossip-data/tags/golden/page/0/" />
    <meta property='og:title' content="golden Page 0" />
    
    <meta name="description" content="Illustrated pages tagged with golden" />
    <meta name='twitter:description' content="Illustrated pages tagged with golden" />
    <meta property='og:description' content="Illustrated pages tagged with golden" />
  
    
    <meta property='og:image' content="http://zooniverse-static.s3.amazonaws.com/www.sciencegossip.org/subjects/thumb/5527af4ba092de04cc000451.jpg" />
    <meta name='twitter:image' content="http://zooniverse-static.s3.amazonaws.com/www.sciencegossip.org/subjects/thumb/5527af4ba092de04cc000451.jpg" />
  
    <meta name="twitter:card" content="summary" />
    <meta name="twitter:creator" content="@the_zooniverse" />
    <meta name="twitter:site" content="@the_zooniverse" />
    <title>golden Page 0</title>
  
  <link rel="stylesheet" href="https://unpkg.com/open-props"/>
  <link rel="stylesheet" href="https://unpkg.com/open-props/normalize.min.css"/>
  <link rel="stylesheet" href="/css/main.css" />
  </head>
  <body>
    <h1>golden</h1>
<nav aria-label="Page">
  <ul>
  <li>
      <a
      class="current"
        href="../0/"
      >
        0
      </a>
    </li>
  </ul>
</nav>
<ul class="thumbnails">

  <li>
    <a href="../../../../groups/fieldnaturalistr/subjects/ASC0002lzt">
      
    <img
      loading=lazy
      
      alt="Subject ASC0002lzt"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/5527af4ba092de04cc000451.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/gardenersmagazin2/subjects/ASC0002o72">
      
    <img
      loading=lazy
      
      alt="Subject ASC0002o72"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/5527af72a092de04cc0019ed.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/the-journal-of-horticulture-cottage-gardener-and-country-gentlemen/subjects/ASC0003mn6">
      
    <img
      loading=lazy
      
      alt="Subject ASC0003mn6"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/56c48a2dbcdc200042002687.jpg"
    />
  
    </a>
  </li>

  <li>
    <a href="../../../../groups/the-journal-of-horticulture-cottage-gardener-and-country-gentlemen/subjects/ASC0003mn6">
      
    <img
      loading=lazy
      
      alt="Subject ASC0003mn6"
      src="https://thumbnails.zooniverse.org/150x150//www.sciencegossip.org/subjects/thumb/56c48a2dbcdc200042002687.jpg"
    />
  
    </a>
  </li>

</ul>
<nav aria-label="Page">
  <ul>
  <li>
      <a
      class="current"
        href="../0/"
      >
        0
      </a>
    </li>
  </ul>
</nav>
  </body>
</html>